integration-tests/features/stack_analyses_npm_ecosystem.feature

```diff
@@ -0,0 +1,126 @@
+Feature: Thorough stack analysis v3 API tests - NPM ecosystem
+
+
+  Scenario: Check that the API entry point requires authorization token
+    Given System is running
+    When I send NPM package manifest package.json to stack analysis version 3 without authorization token
+    Then I should get 401 status code
+
+
+  @requires_authorization_token
+  Scenario: Check the stack analysis response when called with proper authorization token
+    Given System is running
+    When I acquire the authorization token
+    Then I should get the proper authorization token
+
+    # request the stack analysis
+    When I send NPM package manifest package.json to stack analysis version 3 with authorization token
```

Comment:
We can randomly pick some stack here. WDYT?

Reply:
yes, I'm going to create PR with required changes in tests. The "engine" would need to be updated to support this.